.github/workflows/check-links.yaml

```diff
@@ -39,15 +29,12 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Install doc dependencies
         run: make install-site-deps
-      - name: Build docs for link check
+      - name: Build docs
         run: make build-pages
-        # Using liche action to check generated HTML site
-      - name: Link Checker (generated site)
-        id: lc
-        uses: peter-evans/link-checker@v1
+      - name: Check links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
Can anyone tell me why we were using two link checkers?

Reply:
does seem weird, they both came in with the same commit so it looks deliberate. do they each offer different features?

Reply:
yeah, I think it has to do with checking `userdocs` as HTML and other `.md` files as Markdown...

Reply:
leaving it for now